pr-deploy/src/app.ts

```diff
@@ -70,7 +76,8 @@ function initializeRouter(app: Application) {
     response.send({status: 200});
   };
 
-  router.post('/headshas/:headSha/:result', initialize);
+  router.post('/headshas/:headSha/:result(success)/:externalId', initialize);
+  router.post('/headshas/:headSha/:result(errored|skipped)', initialize);
```

Comment:
Why add this line? No need to initialize when there are no build files?

Reply:
`initialize` is the method that handles all three cases, I'm just splitting between the case that now requires the `externalId` and the two other cases that don't